worker/upload_test.go

```diff
@@ -0,0 +1,276 @@
+package worker
```

Comment:
At this point it might make sense to have a dedicated `mocks_test.go` file.

Reply:
ok